streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -642,6 +642,30 @@ public static void addAvgAndMaxToSensor(final Sensor sensor,
         );
     }
 
+    public static void addMaxAndMinToSensor(final Sensor sensor,
```

Comment:
prop (super-nit): Could you call the method `addMinAndMaxToSensor()` since we have already one method that is called `addAvgAndMinAndMaxToSensor()`?